.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "\U0001F680Feature request"
+about: Suggest a feature for webhint
+
+---
+<!--
+
+Hi there 👋
+
+Before opening a new issue please make sure to search in the existing
```

Comment:
How about an opener something like "We're excited for your ideas about how to make webhint even better! Before opening a new issue..."